easybuild/easyconfigs/g/GST-plugins-bad/GST-plugins-bad-1.19.2-GCC-11.3.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'MesonNinja'
+
+name = 'GST-plugins-bad'
+version = '1.19.2'
```

Comment:
There is a 1.20.2 for this as well: https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.20.2.tar.xz

Reply:
hmm, somehow overlooked that

Reply:
@branfosj bumped to 1.20.2